e2e/oversubscription/testdata/exec.nomad

```diff
@@ -0,0 +1,50 @@
+job "oversubscription-exec" {
+  datacenters = ["dc1"]
+
+  constraint {
+    attribute = "${attr.kernel.name}"
+    value     = "linux"
+  }
+
+  group "group" {
+    task "task" {
+      driver = "exec"
+
+      config {
+        command = "/bin/sh"
+        args    = ["-c", "cat /proc/self/cgroup  | grep memory | cut -d: -f3 | tee ${NOMAD_ALLOC_DIR}/tmp/cgroup_name; sleep 1000"]
+      }
+
+      resources {
+        cpu        = 500
+        memory     = 20
+        memory_max = 30
+      }
+    }
+
+    task "cgroup-fetcher" {
```

Comment:
Exec doesn't mount cgroup filesystem into the exec container - so I needed to have this raw_exec "sidecar" to lookup the relevant cgroup and memory limit instead.

Reply:
Yes, not sure why we didn't. Exec is hopelessly behind other drivers in goodness - it may make sense to combine all of that in a exec-v2 refresh.

Reply:
Fair.